tests/lib/collectors/gsheets.py

```diff
@@ -76,21 +49,41 @@ def testValidateSpreadSheetId(self):
         '1DD78vj61BEBoqpw69EdOoaxBUdDqM1GFxk5qRj7-vr4')
 
   @mock.patch('dftimewolf.lib.collectors.gsheets.discovery')
-  def testParseSheet(self, _mock_discovery):
-    """Tests that the collector can parse valid and returns None for invalid sheets."""
+  def testExtractEntiresFromSheet(self, _mock_discovery):
+    """Tests that the collector can extract entries from a valid sheet and
+      returns None for invalid sheet if validate_columns is True. And that it
+      can extract entries from both valid and invalid sheets if validate_columns
+      is False.
+    """
     test_state = state.DFTimewolfState(config.Config)
     collector = gsheets.GoogleSheetsCollector(test_state)
+
     spreadsheet_id = '1DD78vj61BEBoqpw69EdOoaxBUdDqM1GFxk5qRj7-vr4'
     sheet_title = 'Sheet1'
 
     service = _mock_discovery.build.return_value
+
+    # Testing with column validation is True
+    collector.SetUp(spreadsheet_id, sheet_title, True)
+
+    service.spreadsheets.return_value.values.return_value.get.return_value.execute.return_value = VALID_SHEET
+
+    self.assertIsNotNone(collector._ExtractEntiresFromSheet(spreadsheet_id, sheet_title))
+
+    service.spreadsheets.return_value.values.return_value.get.return_value.execute.return_value = INVALID_SHEET
+
+    self.assertIsNone(collector._ExtractEntiresFromSheet(spreadsheet_id, sheet_title))
```

Comment:
```suggestion
    service.spreadsheets.return_value.values.return_value.get.return_value.execute.return_value = VALID_SHEET
    self.assertIsNotNone(collector._ExtractEntiresFromSheet(spreadsheet_id, sheet_title))

    service.spreadsheets.return_value.values.return_value.get.return_value.execute.return_value = INVALID_SHEET
    self.assertIsNone(collector._ExtractEntiresFromSheet(spreadsheet_id, sheet_title))
```

Reply:
I think removing the newlines makes for a more readable test, highlighting "change mock" / assert pairs. This also applies to the codeblock below (L80-86)

Reply:
Grouped them and removed unneeded new lines.